tcp/clientconn.go

```diff
@@ -132,11 +132,16 @@ func bwCreateReleaseMessage(messagePool *pool.Pool) func(m blockwise.Message) {
 	}
 }
 
+type observationMessage struct {
+	ctx context.Context
+	message.Message
+}
+
 func bwCreateHandlerFunc(messagePool *pool.Pool, observationRequests *kitSync.Map) func(token message.Token) (blockwise.Message, bool) {
```

Comment:
Is this on the latest master? I think I've removed almost all usage of `kitSync` pkg.

Reply:
Fixed